.gitignore

```diff
@@ -66,4 +66,8 @@ node_modules
 
 .vscode
 
-scripts
\ No newline at end of file
+scripts
+
+.DS_Store
```

Comment:
.DS_Store is still visible. So what I would do is delete the .DS_Store file that you have and then commit. This should remove the file from GitHub, and you will not lose any data by deleting it locally.